ocaml/xapi/xapi_session.ml

```diff
@@ -1195,6 +1196,13 @@ let local_logout ~__context =
   let session_id = Context.get_session_id __context in
   Xapi_local_session.destroy ~__context ~self:session_id
 
+let check_uname uname = 
```

Comment:
Better to rename as `assert_uname`

Reply:
All the comments have been fixed in the latest commit, Thanks.